Library/Homebrew/shims/super/cc

```diff
@@ -267,6 +285,24 @@ class Cmd
     args
   end
 
+  def ldflags_linux(args)
+    wl = "-Wl," unless mode == :ld
```

Comment:
`if mode != :ld`. Can this also be combined with the one a couple of lines below?

Reply:
Done.